airbyte-integrations/connectors/destination-kinesis/src/main/resources/spec.json

```diff
@@ -8,7 +8,14 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Kinesis Destination Spec",
     "type": "object",
-    "required": ["shardCount", "accessKey", "privateKey", "bufferSize"],
+    "required": [
+      "endpoint",
+      "region",
+      "shardCount",
+      "accessKey",
+      "privateKey",
+      "bufferSize"
+    ],
     "additionalProperties": true,
     "properties": {
       "endpoint": {
```

Comment:
I think `region` could be inferred from `endpoint`: 
e.g `kinesis.us-east-2.amazonaws.com`

Reply:
It'd be great to have examples for required fields.